models/balancer/ethereum/balancer_v2_ethereum_pools_tokens_weights.sql

```diff
@@ -0,0 +1,51 @@
+{{
+    config(
+        schema = 'balancer_v2_ethereum',
+        alias='pools_tokens_weights',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "balancer",
+                                    \'["metacrypto", "jacektrocinski"]\') }}'
```

Comment:
Same as in v2_arbitrum.
```suggestion
        alias='v2_pools_tokens_weights',
        materialized = 'incremental',
        file_format = 'delta',
        incremental_strategy = 'merge',
        unique_key = ['pool_id', 'token_address'],
        post_hook='{{ expose_spells(\'["ethereum"]\',
                                    "project",
                                    "balancer",
                                    \'["metacrypto", "jacektrocinski"]\') }}'
```

Reply:
Done.